composer.json

```
@@ -99,14 +99,14 @@
         "symfony/http-foundation": "4.4.*",
         "symfony/http-kernel": "4.4.*",
         "symfony/lock": "4.4.*",
+        "symfony/mailer": "^4.4 || 5.1.*",
```

```suggestion
        "symfony/mailer": "4.4.*",
```

This is still wrong. We are not yet supporting Symfony 5, therefore please use the same version constraint as we do for all other Symfony packages.

---

But we could support individual components of Symfony 5, in my opinion.

---

Maybe. We can leave the PR open until we have discussed it or we can adjust it and merge it now. 🤷‍♂️

---

I'd rather have it merged now, so that we can move on to https://github.com/contao/contao/pull/1830 😁 . Changed in aa46267